src/components/manage/Blocks/Search/hocs/withSearch.jsx

```diff
@@ -282,34 +282,49 @@ const withSearch = (options) => (WrappedComponent) => {
     const timeoutRef = React.useRef();
     const facetSettings = data?.facets;
 
+    const deepQuery = JSON.stringify(data.query);
     const onTriggerSearch = React.useCallback(
-      (toSearchText, toSearchFacets, toSortOn, toSortOrder) => {
+      (
+        toSearchText = undefined,
```

Comment:
Adding the undefined is not mandatory, but I find it more readable as it tells the developers that they are expected to be undefined in some call. (The bug I fixed was caused by the code not being prepared for undefined values.)